src/Donations/Micros/PaymentStatus/ThankyouMessage.tsx

```
@@ -41,7 +45,10 @@ function ThankyouMessage({
   // EXAMPLE: We've sent an email to Sagar Aryal about the gift.
   // TO DO - if recipientEmail is not present, then show message - You will receive the Gift certificate for {{recipientName}} on your email
   const donationGiftMessage =
-    donation && donation.gift && donation.gift.recipientEmail //TODOO - address TS warnings after /donations is updated to send gift data
+    donation &&
+    donation.gift &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)